website/content/api-docs/auth/aws.mdx

```diff
@@ -56,6 +57,14 @@ capabilities, the credentials are fetched automatically.
   must have the permissions described here.
 - `secret_key` `(string: "")` - AWS Secret key with permissions to query AWS
   APIs.
+- `role_arn` `(string: "")` – <EnterpriseAlert product="vault" inline /> Role ARN to assume
+  for plugin workload identity federation. Required with `identity_token_audience`.
+- `identity_token_audience` `(string: "")` - <EnterpriseAlert product="vault" inline /> The
```

Comment:
We should also call out that `access_key` is mutually exclusive with this field on L48